src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetail.component.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react'
+import EventsComponent from './NodeDetailTabs/Events.component';
+import LogsComponent from './NodeDetailTabs/Logs.component';
+import ManifestComponent from './NodeDetailTabs/Manifest.component';
+import TerminalComponent from './NodeDetailTabs/Terminal.component';
+import './nodeDetail.css';
+import SummaryComponent from './NodeDetailTabs/Summary.component';
+import { NavLink, Route, Switch } from 'react-router-dom';
+import { useParams, useRouteMatch, useHistory } from 'react-router';
+import { NodeDetailTab } from './nodeDetail.type';
+import { getNodeDetailTabs } from './nodeDetail.util';
+import { NodeType } from '../../appDetails.type';
+import AppDetailsStore from '../../appDetails.store';
+import { URLS } from '../../../../../config';
+
+function NodeDetailComponent() {
+
+    const params = useParams<{ actionName: string, podName: string, nodeType: string }>()
+    const [tabs, setTabs] = useState([])
+    const [selectedTabName, setSelectedTabName] = useState("")
+    const { path, url } = useRouteMatch()
+    const history = useHistory()
+
+    useEffect(() => {
+        if (params.nodeType) {
+            const _tabs = getNodeDetailTabs(params.nodeType as NodeType)
+            setTabs(_tabs)
+        }
+
+    }, [params.nodeType, params.podName])
```

Comment:
this should never be reloaded for tab, we should not use [params.nodeType, params.podName] but rather [] also we are not using podName in this useEffect